src/lib/services/email-service.ts

```diff
@@ -537,14 +537,11 @@ export class EmailService {
         },
     ): Promise<IEmailEnvelope> {
         if (this.configured()) {
-            const unsubscribeUrl = '{{amazonSESUnsubscribeUrl}}'; // FIXME: Add unsubscribe URL
```

Comment:
We will probably need to also put unsubscripe url in headers. We need it on this level

Reply:
But it will be different from the profile link that I used and composed in the template. I will add a task for the header in issue tracking